css-brochure-form/styles.css

```diff
@@ -0,0 +1,58 @@
+form {
+  width: 50%;
+  margin: auto;
+}
+
+h3 {
+  color: #ec5d58;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
+  text-align: center;
+}
+
+label {
+  padding-left: 1.5%;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 350;
+  src: url("BrandonGrotesque-Regular.woff2") format("woff2");
```

Comment:
You don't need these, they are only for the `@font-face` rules we provided for you. In order to apply the font family, you only need to use the `font-family` property.